File: ase/misc/clr/ex.js

// var WShell = COM.CreateObject("WScript.Shell");
var Forms = CLR.Load("System.Windows.Forms");
var WebBrowser = Forms.GetType("System.Windows.Forms.WebBrowser");

var on_resize = { };
var mouse_click = { };
var wb = WebBrowser();
CLR.AddEventHandler(wb, "Resize", on_resize);
for (var i = 0; i < 1000; ++i) {
  try {
  CLR.AddEventHandler(wb, "MouseClick", mouse_click);
  } catch (e) {
  }
}

File: ase/misc/clr/ex.lua

-- local WShell = COM.CreateObject("WScript.Shell");
local Forms = CLR.Load("System.Windows.Forms");
local WebBrowser = Forms.GetType("System.Windows.Forms.WebBrowser");

local wb = WebBrowser();
CLR.AddEventHandler(wb, "Resize", on_resize);
function f() 
  CLR.AddEventHandler(wb, "MouseClick", mouse_click);
end
local err, r;
local i;
for i = 1, 100 do
  r, err = pcall(f);
  print(err);
end

